README.md

```diff
@@ -1,39 +1,30 @@
-# PG2: A *Fast* PostgreSQL Driver For Clojure
+# PG2: *Fast* PostgreSQL driver for Clojure
 
-[pg]: https://github.com/igrishaev/pg
+`PG2` is a JDBC-free PostgreSQL client library.
```

Comment:
maybe you can come up with some description which you use in both title here and repo title